extension/content.js

```diff
@@ -207,6 +207,58 @@ function indentInput(el, size = 4) {
 	el.selectionEnd = el.selectionStart = selectionStart + indentationText.length;
 }
 
+function lineUpInput(el, down) {
+	el.focus();
+	const value = el.value;
+	const selectionStart = el.selectionStart;
+	const selectionEnd = el.selectionEnd;
+	const lines = value.split(/(\r?\n)/);
```

Comment:
Oh, I didn't realize it did that.

This is a comment box though. I don't really see the point of preserving the newline character.